lib/taiko.js

```diff
@@ -896,7 +896,8 @@ module.exports.title = title;
  * await click('Get Started', { navigationTimeout: 60000,  force: true })
  * @example
  * await click('Get Started', { navigationTimeout: 60000 }, below('text'))
- *
+ * @example
+ * await click('Get Started', { navigationTimeout: 60000, position: 'right' }, below('text'))
```

Comment:
I noticed that there's nothing added for this new option at 

https://github.com/getgauge/taiko/blob/310097c072d82ba791db7f044d779340d7ed55ca/types/taiko/index.d.ts#L56-L60